pom.xml

```diff
@@ -135,6 +140,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>narayana-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>tomcat-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
```

Comment:
+1 - I can do a release of Narayana once we all agree that is all that is outstanding